iceoryx_posh/test/moduletests/test_popo_toml_gateway_config_parser.cpp

```diff
@@ -204,7 +204,9 @@ TEST_F(TomlGatewayConfigParserSuiteTest, NoServicesInConfigReturnIncompleteConfi
     EXPECT_EQ(TomlGatewayConfigParseError::INCOMPLETE_CONFIGURATION, result.get_error());
 }
 
-TEST_F(TomlGatewayConfigParserSuiteTest, ParseWithoutParameterTakeDefaultPathReturnNoError)
+// Without argument the iceoryx default config in /etc/iceoryx/gateway_config.toml is used. Then this
+// test fails on every machine which is using such a config.
```

Comment:
Why not just remove the default parameter and just pass it on the caller side?

Reply:
This would break the current API where the default config path is always used when no one would specify it otherwise.
I did not want to break the API in this PR but I created an issue: #908